worker/src/test/scala/org/apache/celeborn/service/deploy/worker/storage/DeviceMonitorSuite.scala

```diff
@@ -357,7 +357,8 @@ class DeviceMonitorSuite extends AnyFunSuite {
       device1.notifyObserversOnNonCriticalError(mountPoints1, DiskStatus.READ_OR_WRITE_FAILURE)
       device1.notifyObserversOnNonCriticalError(mountPoints1, DiskStatus.IO_HANG)
       val deviceMonitorMetrics =
-        workerSource.gauges().filter(_.name.startsWith("Device")).sortBy(_.name)
+        workerSource.gauges().filter(_.name.startsWith("Device_" + device1.deviceInfo.name))
```

Comment:
DeviceMonitorSuite ut not sucess but not cause check failed, may be affected by mvn process exit.